metadata-ingestion/src/datahub/ingestion/source/sql/mssql/job_models.py

```diff
@@ -211,6 +212,14 @@ def as_datajob_info_aspect(self) -> DataJobInfoClass:
             status=self.status,
         )
 
+    @property
+    def as_subtypes_aspect(self) -> SubTypesClass:
+        assert isinstance(self.entity, (JobStep, StoredProcedure))
+        type = "Job Step" if isinstance(self.entity, JobStep) else "Stored Procedure"
```

Comment:
ideally we keep all subtype options in the enum in https://github.com/datahub-project/datahub/blob/ac13f255e6f2dd6fc95e2d25ff610b842a2d0aa9/metadata-ingestion/src/datahub/ingestion/source/common/subtypes.py#L0-L1

Reply:
addressed in adb96791a2b25847faf2e0592de46cc5688a5c98